### PR TITLE
Task/jenkins 46866 jira integration module  for 1.2 release

### PR DESCRIPTION
**Pull request by vivek**
# Description

See [JENKINS-46866](https://issues.jenkins-ci.org/browse/JENKINS-46866).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-jira/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>blueocean-parent</artifactId>
+        <groupId>io.jenkins.blueocean</groupId>
+        <version>1.3.0-beta-3-SNAPSHOT</version>
```
Comment (michaelneale) [inline blueocean-jira/pom.xml]: :bug: this needs to be 1.2 right?

Comment (vivek) [reply blueocean-jira/pom.xml]: Ah, fixed!

**Comment by michaelneale** [comment]
```
[ERROR] /home/ubuntu/slave/workspace/jira-integration-module-1.2-KPX7VZH62I5AWW2IO3RYUOFBZJYRJHIE7XOYRTZ345PWTGVWAL2A/blueocean-jira/src/test/java/io/jenkins/blueocean/service/embedded/jira/JiraSCMListenerTest.java:[7,20] package hudson.maven does not exist
[INFO] 1 error
```

**Comment by michaelneale** [comment]
as long as this passes this seems ok @vivek

**Pull request merged**